packages/widgets/src/blocks/legacy-widget/editor.scss

```diff
@@ -164,11 +164,12 @@
 }
 
 .wp-block-legacy-widget {
-	.components-base-control {
+	.wp-block-legacy-widget__selector-container {
 		width: 100%;
```

Comment:
This override was added because of flex on the `Placeholder` fieldset, with no options on `Placeholder` to change that other than with CSS. So I added a div with a class to specify the style of this block and remove it from `BaseControl`. I could have added the class to `WidgetTypeSelector`, but it would be more code changes in two different files, so it seemed better to go this way.